### ISSUE TITLE
shifty gene name case

### ISSUE THREAD

**Issue by pstoilov** [wencke/wencke.github.io#2]
In the circle_dat function all gene names get converted to upper case (line 56). I am assuming that the goal is to make string comparisons more reliable. However, this creates a problem in the chord_dat function which does not change the case of the gene names in the "genes" vector/data frame.  This causes the string comparisons to fail whenever the original gene names set has lower case letters. When dealing with human genes named following standard nomenclature (all upper case) this not a problem. In contrast mouse gene naming capitalizes only the first letter of the gene and will brake chord_dat on lines 42 or 53. I would suggest to keep the case of the data as is and change the case only temporary for the purpose of comparison. This wold also save frustration when preparing publication figures which need to adhere to the gene naming conventions.

Let me know if you want me to fix that for you.

**Comment by wencke** [comment]
Yes, I've already noticed this and I'm currently working on it. Thanks
